Sources/PointFree/Admin/FreeEpisodes.swift

```diff
@@ -0,0 +1,111 @@
+import Css
+import Either
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Optics
+import Prelude
+import Styleguide
+import Tuple
+
+let indexFreeEpisodeEmailMiddleware =
```

Comment:
all the stuff in this file is very similar to what we do for new episode emails, so I could prob combine a lot of this code later.